query/parse.go

```diff
@@ -154,6 +155,13 @@ func parseExpr(in []byte) (Q, int, error) {
 	case tokBranch:
 		expr = &Branch{Pattern: text}
 	case tokText, tokRegex:
+		if !enableRegex {
+			for len(text) > 1 && text[0] == '(' && text[len(text)-1] == ')' {
+				text = text[1 : len(text)-1]
```

Comment:
could there be spaces? so `(    (abvc) )`? so we should trim text after before/after cutting brackets?

Reply:
After this change in UI, queries will work without removal of ( ) which I added earlier. Hence, removed this code altogether.